src/main/scala/com/peoplepattern/text/LangBundle.scala

```diff
@@ -144,24 +170,57 @@ trait LangBundle {
     termNgrams.toSet
   }
 
+  /**
+   * Extract the set of term-only bigrams from the text
+   *
+   * For example from the text "this is the winning team" only the bigram
+   * "winning team" would be extracted
+   *
+   * @param text the text to extract n-grams from
+   */
   def termBigrams(text: String) = termNgrams(text, 2, 2)
 
+  /**
+   * Extract the set of term-only bigrams from the token sequence
+   *
+   * For example from the text "this is the winning team" only the bigram
+   * "winning team" would be extracted
+   *
+   * @param tokens the token sequence to extract n-grams from
+   */
   def termBigrams(tokens: Seq[String]) = termNgrams(tokens, 2, 2)
 
+  /**
+   * Extract the set of term-only bigrams from the text
+   *
+   * For example from the text "this is red sox nation" only the trigram
+   * "red sox nation" would be extracted
+   *
+   * @param text the text to extract n-grams from
+   */
   def termTrigrams(text: String) = termNgrams(text, 3, 3)
 
+  /**
+   * Extract the set of term-only bigrams from the text
+   *
+   * For example from the text "this is red sox nation" only the trigram
+   * "red sox nation" would be extracted
+   *
+   * @param tokens the token sequence to extract n-grams from
+   */
   def termTrigrams(tokens: Seq[String]) = termNgrams(tokens, 3, 3)
 }
 
+/** Helpers and language-specifi [[LangBundle]]s **/
 object LangBundle {
 
   import scala.io.Source
 
-  def srcFromResource(path: String) = {
+  private def srcFromResource(path: String) = {
     Source.fromInputStream(getClass.getResourceAsStream(path))
```

Comment:
Not exactly not-thread-safe, but you're right this doesn't work in some threaded environments. Taking this up in #11.